src/frequenz/sdk/microgrid/client.py

```diff
@@ -24,6 +24,9 @@
 from .connection import Connection
 from .retry import LinearBackoff, RetryStrategy
 
+# Default timeout applied to all gRPC calls
+DEFAULT_GRPC_CALL_TIMEOUT = 60.0
```

Comment:
I thought we had decided to make this an instance variable of `MicrogridGrpcClient` in order to make it configurable.  Wasn't that the decision?

Reply:
hmm it depends. 
For example for the `set_power` call I use asyncio.wait with timeout anyway - because this is too big.

If we plan to use this timeout as default (preferable) then it would be better to make it constructor parameter and smaller.
But if we don't plan this, then this timeout is only to be extra safe. And there is no reason to make it configurable.